test/osutils.jl

```diff
@@ -56,3 +57,15 @@ if Sys.iswindows()
         end
     end
 end
+
+if Sys.islinux() && Sys.which("readelf") !== nothing
+    @testset "stack is not marked as executable" begin
+        for f in intersect(dllist(),
+                           [readdir(joinpath(Sys.BINDIR, Base.LIBDIR), join=true);
+                            readdir(joinpath(Sys.BINDIR, Base.LIBDIR, "julia"), join=true)])
```

Comment:
I guess this check is too cautious as it shouldn't have passed on 64-bit Linux until the blastrampoline PR is merged. Maybe the paths are different so the intersection ends up being empty? We could just use `dllist()`, the risk that a system library we link to would have executable stack enabled is quite low. The other approach would be to just keep the `readdir` part, but I figured it could be annoying as unused outdated libjulia.so.1.X can be in these directories (I had some), which would make tests fail and require manual removal (which is hard to find out).